src/agents/gc/README.md

```diff
@@ -1,22 +1,22 @@
 # Prototype of an OSConfig Native Resource Provider (NRP) for Azure Automanage Machine Configuration (MC) 
 
-## About Machine Configuration
+## 1. About Machine Configuration
 
 Read about Azure Automanage Machine Configuration (formerly called Azure Policy Guest Configuration) at [learn.microsoft.com/machine-configuration](https://learn.microsoft.com/en-us/azure/governance/machine-configuration/).
 
-## Regenerating the NRP code for a changed resource class:
+## 2. Regenerating the NRP code for a changed resource class
 
 To regenerate code, see [codegen.cmd](codegen.cmd). Warning: regenerating code will overwrite all customizations and additions specific to OSConfig. 

```

Comment:
Can we use the warning blockquote?
See https://github.com/orgs/community/discussions/16925 for more info

> **Warning**
> This is a warning